docs/_includes/common/bundling.html

```diff
@@ -1,7 +1,43 @@
 <div>
     <h2 id="bundling">Bundling</h2>
     <p>Skin supports <a href="https://webpack.js.org">Webpack</a> and <a href="https://github.com/lasso-js/lasso">Lasso.js</a>.</p>
-    <p>For Lasso.js, add the <span class="highlight">@ebay/skin</span> module as a <span class="highlight">browser.json</span> dependency to include the entire set of Skin modules.</p>
+
+    <h3>Webpack</h3>
+    <p>Add <span class="highlight">@ebay/skin</span>, <span class="highlight">@ebay/browserslist-config</span> as dependencies and <span class="highlight">less</span>, <span class="highlight">less-loader</span> as dev dependencies to <span class="highlight">package.json</span>.</p>
```

Comment:
We don't need to include `@ebay/browserslist-config` as a dependency, it is optional.

The app consuming this via webpack would need to have `webpack-cli` or similar installed though.

Reply:
It forced me to install `@ebay/browserslist-config` when I created a sample app with webpack